Project_Management_ELCA/Project_Management_ELCA/src/main/java/pilotproject/Project_Management_ELCA/entity/Employee.java

```diff
@@ -0,0 +1,61 @@
+package pilotproject.Project_Management_ELCA.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author gtn
+ *
+ */
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+//@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String visa;
+
+    @Column(nullable = false)
+    private String first_name;
+
+    @Column(nullable = false)
+    private String last_name;
+
+    @Column(nullable = false)
+    private Date birth_date;
+
+    @Column(nullable = true)
+    private int version;
+
+    @OneToOne(mappedBy = "employee")
+    private Group group;
+
+    @ManyToMany(mappedBy = "employeeList")
+    private Set<Project> projectList;
```

Comment:
[S] In the context of this project, it is much easier if we define this relationship in `Project` instead of `Employee`